server/config/config.template.yaml

```diff
@@ -40,6 +40,11 @@ server:
   ### Public URL. It is used in welcome.html if not configured it will be taken from 'Host' http header on welcome.html requests
   #public_url: https://yourhost #Optional.
 
+  ### Limit the count of columns for any destinations.
+  ### It helps to prevent generating an infinite amount of columns in databases.
+  ### Zero value turns off column count calculation.
+  max_columns: 100 #Optional
```

Comment:
I think it would be better to have this flag on the destination configuration level (in `data_layout` section). 
```
destinations:
  pg:
   ...
    data_layout:
      max_columns: 100
```
The logic might be:
- if a destination doesn't have configured `max_columns` - apply default value = `100`

Reply:
It was already mentioned at the [issue](https://github.com/jitsucom/jitsu/issues/140#issuecomment-803409720).
It looks like a simple error checking.
Will it be different for different destinations?
If not - it can be used just for simple checking.

Reply:
Fixed.
To error checking, an option is used on the server level if it is positive.
Any destination can override that setting by any positive value.